### DISCUSSION TITLE
MIT licence in patch publishing options

### DISCUSSION THREAD

**Discussion by TobyKLight** [cables-gl/cables_docs#662] [Feature Requests]
Request is for ability to select MIT licence when publishing a patch. 
Seems odd that cables core is licenced under MIT and then contributions from users are limited to Creative Commons licences? 

Indeed in the documentation it says 
"We prefer the MIT License" 
https://cables.gl/docs/faq/features/contribute/contribute

Possibly also makes sense to have a 'custom licence' field. (So you can describe that your patch has a mix of MIT and creative commons content if it already uses content from others)

**Comment by pandrr** [comment]
good point

the licences are for patches. CC licenses for code are not very useful we think.
probably the ops themselves should have a licence select option...
